mempool/mempool.go

```diff
@@ -32,13 +32,15 @@ func New(db dbm.DB) *Pool {
 	}
 }
 
-func (p *Pool) Enqueue(userTxn comettypes.Tx) error {
+func (p *Pool) Enqueue(userTxn comettypes.Tx) (err error) {
 	// NOTE: we should do reads and writes on the same view. Right now they occur on separate views.
 	// Unfortunately, comet's DB interface doesn't support it.
 	// Moving to a different DB interface is left for future work.
 
 	batch := p.db.NewBatch()
-	defer batch.Close() // TODO: catch error.
+	defer func() {
+		err = batch.Close()
```

Comment:
We should wrap the error, not reset it. Let's wait for #44 to merge before moving forward with this. It contains a helper that can be useful here.

Reply:
Can you provide a link to the helper you wrote about?

Reply:
[`localdb.wrapCloseErr`](https://github.com/polymerdao/monomer/blob/9f7fc59cb4d58e5def9b8c7fbbb686cc0f596d64/monomerdb/localdb/localdb.go#L455)

It will need to be moved to `utils` so it can be used outside of the `localdb` package.